package.json

```diff
@@ -8,19 +8,21 @@
     "dist"
   ],
   "scripts": {
-    "build": "rimraf dist && babel src -d dist",
-    "docs": "gulp docs",
+    "docs": "gulp",
+    "build": "npm run build:dist && npm run build:docs",
+    "build:dist": "rimraf dist && babel src -d dist",
+    "build:docs": "gulp docs",
```

Comment:
`npm run docs` didn't run docs, not intuitive.  Renamed scripts to make more sense.